apps/storybook-react/stories/Button.stories.jsx

```diff
@@ -67,6 +78,29 @@ export const contained = () => (
     <Button color="secondary">Secondary</Button>
     <Button color="danger">Danger</Button>
     <Button disabled>Disabled</Button>
+    <Button leftIcon={<Icon name="save" title="save"></Icon>}>Primary</Button>
```

Comment:
I know this is a common pattern, but we have deliberately avoided it so far in favour of composition. I would do something like this instead: 

```jsx
<Button>
  <Icon name="save" title="save" />
  Primary
</Button>
```